docs/repo/property-pages/property-specification.md

```diff
@@ -190,6 +190,8 @@ If the text within this editor should be displayed with a monospace (fixed width
 </StringProperty>
 ```
 
+Multi-line strings can optionally set metadata to control text wrapping, where the value corresponds to an enum value of [TextWrapping](https://learn.microsoft.com/en-us/dotnet/api/system.windows.textwrapping?view=windowsdesktop-7.0). Example: `<NameValuePair Name="TextWrapping" Value="Wrap" />`. Text-wrapping defaults to NoWrap.
```

Comment:
Don't have `en-us` in the url.
`https://learn.microsoft.com/dotnet/api/system.windows.textwrapping?view=windowsdesktop-7.0`

Reply:
Also use the `netframework-4.7.2` version as that's the version of WPF we target in VS.

```suggestion
Multi-line strings can optionally set metadata to control text wrapping, where the value corresponds to an enum value of [TextWrapping](https://learn.microsoft.com/dotnet/api/system.windows.textwrapping?view=netframework-4.7.2). Example: `<NameValuePair Name="TextWrapping" Value="Wrap" />`. Text-wrapping defaults to `NoWrap`.
```